Documentation/ServiceUtilities/MixedRealityServiceRegistryAndIMixedRealityServiceRegistrar.md

```diff
@@ -0,0 +1,54 @@
+# What are the MixedRealityServiceRegistry and IMixedRealityServiceRegistrar
+
+The Mixed Reality Toolkit has two very similarly named components that perform related tasks: 
+MixedRealityServiceRegistry and IMixedRealityServiceRegistrar. What are they and what purpose does each serve?
+
+## MixedRealityServiceRegistry
+
+The MixedRealityServiceRegistry is the component that contains instances of each registered service
+(core systems and extension services). It MixedRealiityServiceRegistry is implemented as a static C#
+class and is the recommended pattern to use to acquire service instances in application code.
+
+The following snippet demonstrates acquiring an IMixedRealityInputSystem instance.
+
+```
+IMixedRealityInputSystem inputSystem = null;
+
+if (!MixedRealityServiceRegistry.TryGetService<IMixedRealityInputSystem>(out inputSystem))
+{
+    // Failed to acquire the input system. It may not have been registered
+}
+```
+
+## IMixedRealityServiceRegistrar
+
+The IMixedRealityServiceRegistrar is the interface that defines the interface implemented by 
```

Comment:
reword: "interface that defines the interface"

Reply:
fixed